broadcasting.md

```diff
@@ -108,19 +109,31 @@ The [laravel-websockets](https://github.com/beyondcode/laravel-websockets) and [
 <a name="ably"></a>
 ### Ably
 
-If you plan to broadcast your events using [Ably](https://ably.io), you should install the Ably PHP SDK using the Composer package manager:
+If you plan to broadcast your events using [Ably](https://ably.com), you should install the Ably Laravel Broadcaster using the Composer package manager:
 
-    composer require ably/ably-php
+```shell
+composer require ably/laravel-broadcaster
+```
+
+> Deprecated - Use this instead, if you are using pusher-js client to connect with ably.
```

Comment:
I don't know if it's a great idea to mark the existing setup as deprecated. Maybe it's fine but I think just a paragraph or two at the top saying that the recommended way to use ably with laravel is to use ably/laravel-broadcaster and linking out to our own documentation.

Reply:
Okay, this will make sense 👍